src/util/ChartUtils.ts

```diff
@@ -574,16 +588,18 @@ export const getCoordinatesOfGrid = (ticks: Array<TickItem>, min: number, max: n
  * @return {Array}  Ticks
  */
 export const getTicksOfAxis = (
-  axis: BaseAxisProps & {
-    duplicateDomain?: any;
-    realScaleType?: 'scaleBand' | 'band' | 'point' | 'linear';
-    scale?: any;
-    axisType?: AxisType;
-    ticks?: any;
-    niceTicks?: any;
-    isCategorical?: boolean;
-    categoricalDomain?: any;
-  },
+  axis:
+    | null
+    | (BaseAxisProps & {
+        duplicateDomain?: any;
+        realScaleType?: 'scaleBand' | 'band' | 'point' | 'linear';
+        scale?: any;
+        axisType?: AxisType;
+        ticks?: any;
+        niceTicks?: any;
```

Comment:
Wow, I have never seen this before. Will have to investigate where and how this is used.